resources/integration_test/tests/dbmd_variable_result_sets_function_methods.yml

```diff
@@ -37,6 +37,7 @@ tests:
     row_count: 19
     expected_sql_type: [LONGVARCHAR, LONGVARCHAR, LONGVARCHAR, LONGVARCHAR, INTEGER,
                         LONGVARCHAR]
+    expected_bson_type: [string, string, string, int, string, string]
```

Comment:
I see what happened.  After the change was merged that orders the metadata resultset columns, SQL-703, I regenerated the tests but missed the ones with a skip_reason since the generator also skips them.  I have fixed these now.
Thanks for catching that.